.github/workflows/lint-golangci.yml

```diff
@@ -6,7 +6,10 @@ on:
     paths:
       - 'go.mod'
       - 'go.sum'
+      - 'Makefile'
```

Comment:
I suggest removing the paths, because now all the checks are required to merge a PR, so if you just did a documentation change in a PR, you will still need the checks to run in order to be able to merge the PR